packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -1,75 +1,93 @@
 .content {
   z-index: var(--elevation-tooltip);
   max-width: var(--popover--width);
+  min-width: calc(var(--space-extravagant) * 3.75);
   box-shadow: var(--shadow-base);
+  margin-top: var(--space-small);
   border: var(--border-base) solid var(--color-border);
-  border-radius: var(--radius-base);
-  font-size: var(--typography--fontSize-base);
-  line-height: normal;
+  border-radius: var(--radius-larger);
+  overflow: scroll;
+  overflow-y: auto;
   background: var(--color-surface);
 }
 
 /* CAUTION: Tests for content visibility assume this class' presence hides the content
-  * so please be careful if you change this class name, or the css rules within.
- */
+* so please be careful if you change this class name, or the css rules within.
+*/
 .content.hidden {
   display: none;
   visibility: hidden;
 }
 
+.clearSearch {
+  display: flex;
+  position: absolute;
+  top: 50%;
+  right: var(--space-base);
+  z-index: var(--elevation-tooltip);
+  width: calc(var(--space-small) * 3);
```

Comment:
Fast-follow: `var(--space-small) * 3` is the same output (24px) as `var(--space-large)` so we can do away with these calcs

Reply:
wow Taylor 😵‍💫